runtime/moonbase/src/lib.rs

```diff
@@ -1610,3 +1621,221 @@ mod tests {
 		}
 	}
 }
+
+#[cfg(test)]
+mod fee_tests {
+	use super::*;
+	use frame_support::weights::WeightToFee;
+	use frame_support::{dispatch::GetDispatchInfo, traits::OnFinalize, weights::DispatchInfo};
+	use sp_runtime::traits::Convert;
+
+	fn run_with_system_weight<F>(w: Weight, mut assertions: F)
+	where
+		F: FnMut() -> (),
+	{
+		let mut t: sp_io::TestExternalities = frame_system::GenesisConfig::default()
+			.build_storage::<Runtime>()
+			.unwrap()
+			.into();
+		t.execute_with(|| {
+			System::set_block_consumed_resources(w, 0);
+			assertions()
+		});
+	}
+
+	#[test]
+	fn test_multiplier_can_grow_from_zero() {
```

Comment:
Why are these not in integration tests? At least I would not include them here..Can we move them elsewhere?

Reply:
I'll revisit that. They were copied from substrate a long time ago...

Reply:
Fixed in recent commits (for all runtimes)